src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -275,20 +282,20 @@ local function serializeImpl(t, tTracking, sIndent)
             return "{}"
         else
             -- Other tables take more work
-            local sResult = "{\n"
-            local sSubIndent = sIndent .. "  "
+            local sResult = "{" .. (opts.noWrap and "" or "\n")
```

Comment:
I wonder if it's worth hoisting the "separator string" logic into one place. So something like:

```lua
local open, close, equal, comma = "{\n", "\n}", " = ", ",\n"
-- I'm sure there's more ("[ ", " ] = ", etc...), but hopefully gives the right idea
if ops.noWhiteSpace then
  open, close, equal, comma = "{", "}", "=", ","
end
```

I'd personally also merge the `noWhiteSpace` and `noWrap` options. I guess I'm seeing two use cases for `textutils.serialize`:
 - Want the output to be easy to read (e.g. printing, configuration files).
 - Want the output to be compact (e.g. saving large amounts of data to a file).